sdk/communication/azure-communication-chat/CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Release History
-## 1.0.0-beta.9 (Unreleased)
+## 1.0.0 (2021-06-09)
+### Dependency Updates
```

Comment:
nit; I think we can move this section to the end of this section, users are probably more interested to know about other changes.